.appveyor.yml

```diff
@@ -7,10 +7,18 @@ environment:
   PYTHON: C:\Python38
   SCONS_CACHE_ROOT: "%HOME%\\scons_cache"
   SCONS_CACHE_LIMIT: 1024
+  OPTIONS: "debug_symbols=no verbose=yes progress=no builtin_libpng=yes"
```

Comment:
`builtin_libpng` defaults to true on all platforms already.

Reply:
So yes, you can remove the stray one in `.travis.yml`.

Reply:
Updated PR to remove `builtin_libpng=yes` from `.travis.yml`